charts/clusterapi-resources/templates/KubeadmConfigTemplate.yaml

```diff
@@ -26,6 +26,9 @@ spec:
       files:
       {{- toYaml . | nindent 6 }}
 {{- end }}
+{{- with $.Values.kubeadmConfigSpec.kubeadmConfigTemplateFiles }}
+      {{- toYaml . | nindent 6 }}
+{{- end }}
```

Comment:
I've changed the approach here, to avoid this silly diff in the image:

![Screenshot 2025-04-25 at 15 01 18](https://github.com/user-attachments/assets/82bd162e-3e5e-42a6-9c88-1492e75f2a71)

This triggers errors on ArgoCD because of immutable objects for the control plane (which needs no change In disks, so it's just causing trouble). 

Without this, we will needlessly have to rollout new control nodes as well.

Reply:
I think you meant to comment on  https://github.com/powerhome/helm-charts/pull/53/files#diff-e0d6daef3a5844d7380d61c36776b62dbf8d30b452becb9122c899b71eddb0e1R17-R19 but I understood haha